### PR TITLE
Release 1.8.5: Kafka Pubsub Hotfix

### PR DESCRIPTION
**Pull request by berndverst**
# Description

Release notes for the 1.8.5 Kafka Pubsub hotfix
Pins contrib 1.8.5-rc.1

### CODE REVIEW

**Review by yaron2 — APPROVED** [review]

File: docs/release_notes/v1.8.5.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Dapr 1.8.5
+
+## Fixes an issue in Kafka Pubsub preventing the component from initializing with SALS password auth
+
+### Problem
+
+The Kafka Pubsub component failed to initialize when using SASL password authentication.
+
+### Root cause
```
Comment (ItalyPaleAle) [inline docs/release_notes/v1.8.5.md:9]: Missing "Impact". See example: https://github.com/dapr/dapr/blob/master/docs/release_notes/v1.8.3.md

File: docs/release_notes/v1.8.5.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Dapr 1.8.5
+
+## Fixes an issue in Kafka Pubsub preventing the component from initializing with SALS password auth
```
Comment (mukundansundar) [inline docs/release_notes/v1.8.5.md:3]: SASL

File: docs/release_notes/v1.8.5.md (hunk 1 above)
Comment (berndverst) [reply docs/release_notes/v1.8.5.md:9]: That's an optional section. It's not missing. We didn't always have that section.

**Review by mukundansundar — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

**Comment by berndverst** [comment]
Manually built this on Windows and ran the tests. It passes there, so not sure why CI/CD had issues.

image

**Pull request merged**